nlp/text_comparison.py

```diff
@@ -11,13 +11,13 @@ def compare_two_texts(a, b, length):
             "intervals": first_to_second[0],
             "similarity": round(
                 first_to_second[1] / len(a), 3
-            ),  # float between 0 and 1
+            ) * 100,
```

Comment:
Please if you moved the multiplication in here, please remove the multiplication in tasks, because now its multiplied twice.